engine-tests/src/test/java/org/terasology/i18n/TranslationFormatTests.java

```diff
@@ -54,20 +54,20 @@ public void setup() {
     }
 
     @Test
-    public void testGetAssetName() throws InvalidAssetFilenameException {
+    public void testGetAssetName()  throws InvalidAssetFilenameException {
         assertEquals(new Name("menu"),  format.getAssetName("menu.lang"));
         assertEquals(new Name("menu_pl"),  format.getAssetName("menu_pl.lang"));
     }
 
-    @Test
-    public void testPathMatcher() {
-        assertFalse(format.getFileMatcher().matches(Paths.get("menu.json")));
-        assertFalse(format.getFileMatcher().matches(Paths.get("menu.prefab")));
-
-        assertTrue(format.getFileMatcher().matches(Paths.get("menu.lang")));
-        assertTrue(format.getFileMatcher().matches(Paths.get("menu_pl.lang")));
-        assertTrue(format.getFileMatcher().matches(Paths.get("menu_en-US-x-lvariant-POSIX.lang")));
-    }
+//    @Test
+//    public void testPathMatcher() throws IOException, InvalidAssetFilenameException{
+//        assertFalse(format.getFileMatcher().matches(Paths.get("menu.json")));
+//        assertFalse(format.getFileMatcher().matches(Paths.get("menu.prefab")));
+//
+//        assertTrue(format.getFileMatcher().matches(Paths.get("menu.lang")));
+//        assertTrue(format.getFileMatcher().matches(Paths.get("menu_pl.lang")));
+//        assertTrue(format.getFileMatcher().matches(Paths.get("menu_en-US-x-lvariant-POSIX.lang")));
+//    }
```

Comment:
do we need this test?

Reply:
TranslationFormat still has that Matcher, so I assume so.

Pushed a fix.